docs/source/index.rst

```diff
@@ -29,6 +19,92 @@ Overall, SciKeras aims at being as flexible as Keras while having a clean interf
    advanced
    api
 
+Differences with ``tf.keras.wrappers.scikit_learn``
+---------------------------------------------------
+
+* Full compatibility with the Scikit-Learn API, including grid searches, ensembles, transformers, etc.
+* Support for Keras Functional and Subclassed Models.
+* Support for pre-trained models.
+* Support for dynamically set Keras parameters depending on inputs (e.g. input shape).
+* Support for hyperparameter tuning of optimizers and losses.
+* Support for multi-input and multi-ouput Keras models.
+* Functional `random_state` for reproducible training.
+* Many more that you will discover as you use SciKeras!
+
+
+Transitioning from ``tf.keras.wrappers.scikit_learn``
+-----------------------------------------------------
+
+SciKeras is largely backwards compatible with the existing wrappers. For most cases, you can just change your import statement from:
+
+.. code:: diff
+
+   - from tensorflow.keras.wrappers.scikit_learn import KerasClassifier, KerasRegressor
+   + from scikeras.wrappers import KerasClassifier, KerasRegressor
+
+
+SciKeras does however have some backward incompatible changes:
+
+One-hot encoding of targets for categorical crossentropy losses
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+SciKeras will not longer implicitly inspect your Model's loss function to determine if
+it needs to one-hot encode your target to match the loss function. Instead, you must explicitly
+pass your loss function to the constructor:
+
+.. code:: python
+
+   clf = KerasClassifier(loss="categorical_crossentropy")  # or loss=CategoricalCrossentropy(), etc.
+
+Removal of ``**kwargs`` from fit and predict
```

Comment:
```suggestion
Removal of keyword arguments from fit and predict
```

Reply:
Is this wording accurate? We don't plan on removing keyword arguments in the sense that we would only user or allow positional arguments. Maybe something like `Removal of arbitrary keyword arguments passed via **kwargs`? But that's pretty long, especially for a title...

Reply:
Why not "Removal of Keras keyword arguments passed to fit/predict"?

Reply:
I'm going to go with `Variable keyword arguments in fit and predict` as the title.